### DISCUSSION TITLE
How to listen to value a and change value b

### DISCUSSION THREAD

**Discussion by beilo** [pmndrs/valtio#576] [Q&A] [answered]
I have an object
```js
let store  = proxy({
  a:null,
  b:2
})
funcation Api(){
  store.a = 1
}
```

I want to monitor the change of a and assign a value to b. What's the best way to deal with it?
thank you.

**Comment by dai-shi** [comment, marked as answer]
You can try `proxyWithComputed` or `derive` from valtio/utils.

**Reply by beilo** [reply]
Thank you very much for your reply

I also need to change b. If you use proxyWithComputed, a new store1 will be added.

I just want to use the store

**Reply by dai-shi** [reply]
You can subscribe to your state and change it. Be careful to avoid infinite loops. subscribeKey may help.
